index.js

```diff
@@ -401,16 +496,22 @@ module.exports = function AsyncAppendOnlyLog(filename, opts) {
     appendTransaction: onLoad(appendTransaction),
     close: onLoad(close),
     onDrain: onLoad(onDrain),
+    compact: onLoad(compact),
     since,
+    compactionSince,
```

Comment:
can we document this in README? It's actually a bit hard on first read to understand what the size number indicates.